modules/client/src/internal/client/methods.ts

```diff
@@ -21,13 +21,15 @@ import { AddressZero } from "@ethersproject/constants";
 import { Contract, ContractTransaction } from "@ethersproject/contracts";
 import { erc20ContractAbi } from "../abi/erc20";
 import {
+  QueryTokenBalances,
```

Comment:
Aren't we calling things like `AssetBalances` before?

Reply:
If there's a difference between subgraph and the SDK, I would suggest the SDK always talks to the exterior world in terms of "Token"

Reply:
Yes, internally is TokenBalances, to the exterioir is AssetBalances

Reply:
Then.... why don't we unify this? What's the benefit of having both?